src/views/parquet_reader.rs

```diff
@@ -143,19 +143,32 @@ impl ParquetUnresolved {
                 self.object_store_url
             );
         }
+
+        let url_hash = self
+            .object_store_url
+            .as_str()
+            .replace("://", "_")
+            .replace('/', "")
+            .replace('-', "_");
+        let registered_table_name = format!("{}_{}", self.table_name.as_str(), url_hash); // The unique name for registration in DataFusion 
```

Comment:
The **registered name** comes from the Objectstoreurl. But need the replacement due to SQL grammar